### ISSUE TITLE
got CA cert error when updating the build

### ISSUE THREAD

**Issue by cloudslsw** [RPCS3/rpcs3#13005]
### Quick summary

issue occur from the beginning of the years and i only can manually update from the website, i didn't delete the emulator at all to try again from fresh start because it contain my HDD games and configs.
the log says:
E NET: curl_easy_perform(): Curl error (77): Problem with the SSL CA cert (path? access rights?)
Details: CA signer not available for verification

### Attach a log file

[RPCS3.log](https://github.com/RPCS3/rpcs3/files/10137284/RPCS3.log)

**Comment by Risusama** [comment]
I believe this does not belong here (help is given in Discord). Anyway, here's what to do to fix it.

Your RPCS3 installation is missing the CA cert file (cacert.pem) in the RPCS3 directory.
When getting RPCS3, you must extract ALL the files from the archive into the target directory, in your case "E:\game\rpcs3".

Failure to do so, for example by only extracting the exe, will result to this error.
The file "cacert.pem" is required for the emulator to verify the update is being downloaded from real RPCS3 website and not from a fake.

TL;DR:
Download latest build and extract ALL THE FILES from the archive to desired folder. Do NOT SKIP any files.

**Comment by cloudslsw** [comment]
> I believe this does not belong here (help is given in Discord). Anyway, here's what to do to fix it.
> 
> Your RPCS3 installation is missing the CA cert file (cacert.pem) in the RPCS3 directory. When getting RPCS3, you must extract ALL the files from the archive into the target directory, in your case "E:\game\rpcs3".
> 
> Failure to do so, for example by only extracting the exe, will result to this error. The file "cacert.pem" is required for the emulator to verify the update is being downloaded from real RPCS3 website and not from a fake.
> 
> TL;DR: Download latest build and extract ALL THE FILES from the archive to desired folder. Do NOT SKIP any files.

thanks for reply, yes, i already did extract all files, but still not working

**Comment by cloudslsw** [comment]
today i delete all files in the rpcs3 folder except <config><dev_hdd0><patches>, and downloaded latest rpcs3 build and extracted, i still got this error

**Comment by oltolm** [comment]
Please retest.

**Comment by cloudslsw** [comment]
> Please retest.

its good now

**Comment by Triticum0** [comment]
Can This be closed then?
